db/blob/blob_index.h

```diff
@@ -26,11 +26,11 @@ namespace ROCKSDB_NAMESPACE {
 //      +------+------------+---------------+
 //
 //    kBlob:
-//      +------+-------------+----------+----------+-------------+
-//      | type | file number | offset   | size     | compression |
-//      +------+-------------+----------+----------+-------------+
-//      | char | varint64    | varint64 | varint64 | char        |
-//      +------+-------------+----------+----------+-------------+
+//      +------+-------------+----------+----------+----------+-------------+
+//      | type | file number | offset   | size     |   etime  | compression |
```

Comment:
etime is too simple to make me confused, expire_time maybe better.

Reply:
done